File: Execicio5_RevisaoPratica/relacionais/relacionais.py
def main():
    try:
        a, b = map(int, input("Digite dois números inteiros (a b): ").split())
    except Exception:
        print("Entrada inválida.")
        return

    # Relacionais
    print("\n--- Relacionais ---")
    print("a > b ?", a > b)
    print("a == b ?", a == b)
    print("Ambos pares?", (a % 2 == 0 and b % 2 == 0))

    
if __name__ == "__main__":
    main()

File: Execício 5 - Revisão Prática/relacionais/relacionais.cs
using System;

class relacionais {
    static void Main() {
        Console.Write("Digite dois números inteiros\n");
        string[] parts = Console.ReadLine()?.Split();
        if (parts == null || parts.Length < 2) {
            Console.WriteLine("Entrada inválida.");
            return;
        }

        int a = int.Parse(parts[0]);
        int b = int.Parse(parts[1]);

        Console.WriteLine($"a > b ? { (a > b) }");
        Console.WriteLine($"a = b ? { (a == b) }");
        Console.WriteLine($"Ambos pares? { (a % 2 == 0 && b % 2 == 0) }");
    }
}


